packages/bits/src/lib/menu/menu-item/menu-option/menu-option.component.less

```diff
@@ -3,26 +3,39 @@
 
 .nui-menu .nui-menu-item, .nui-menu-popup-host .nui-menu-item, .nui-menu-popup .nui-menu-item {
     .nui-menu-item__option {
-        padding: 0 @nui-space-xs;
+        padding: 0;
         .nui-checkbox__label {
             font-weight: @nui-font-weight-regular;
             font-size: @nui-font-size-default;
             width: 100%;
-        }
+            padding: @nui-space-xs+@nui-space-xxs @nui-space-sm+@nui-space-xxs;
 
-        .nui-menu-item__checkbox > .nui-checkbox {
-            padding: 0;
+            &:focus {
+                border-radius: 0;
+                .setCssVariable(background-color, transparent);
+            }
         }
+
+        .nui-menu-item__checkbox {
+            width: 100%;
+                & > .nui-checkbox {
+                    padding: 0;
+                }
+            }
+
         .nui-checkbox__translucent {
             cursor: not-allowed;
         }
     }
 
     &.nui-menu-item--disabled {
         > .nui-menu-item__option {
+            padding-left: @nui-space-xs;
+            padding-right: @nui-space-xs;
```

Comment:
Can be changed to `padding: 0 @nui-space-xs;`

Reply:
Thanks, will be changed